tasks/php_apt_switchversion.yml

```diff
@@ -16,13 +16,14 @@
       -setup
 
   - name: PHP | Fix php family prefix
-    set_fact: php_family_prefix=php
+    set_fact: php_family_prefix=php7
     when: php_family == "default" and (ansible_distribution_release == "xenial") and  php_present|failed
     tags:
       -php
       -setup
 
   - name: PHP | Add apt repository for php 5.4
+    set_fact: php_family_prefix=php5
```

Comment:
you can't combine multiple modules in single step (apt_repository and set_Fact) in that case.
This for 100% should get play error